sundial/gs-sidebar/main.js

```diff
@@ -46,22 +46,47 @@ await front.init(res, ICON_MODEL_URL, QUANTITY_MODEL_URL);
   const downloadCollage = document.querySelector('button.collage');
   const downloadTotals = document.querySelector('button.totals');
   const downloadTSV = document.querySelector('button.tsv');
-  const insertGoogle = document.querySelector('button.insert-google');
+  const insertGoogle = document.querySelector('button.insert-gs');
+  const appendGoogle = document.querySelector('button.append-gs');
 
   front.registerDefaultListeners();
   front.addInputListener(input);
   front.addDownloadCollageListener(downloadCollage);
   front.addDownloadTotalsListener(downloadTotals);
   front.addDownloadTSVListener(downloadTSV);
-  addInsertGoogleListener(insertGoogle);
+  if (insertGoogle !== null) {
+    addInsertGSListener(insertGoogle);
+  }
+  if (appendGoogle !== null) {
+    addAppendGSListener(appendGoogle);
+  }
 })();
```

Comment:
Is this to support both the existing Sundial sheet and the new one?

Reply:
yes. This way rolling back is as simple as just modifying a few lines of html.